guetzli-2017-3-30/test-libfuzzer.sh

```diff
@@ -5,6 +5,7 @@
 set -x
 rm -rf $CORPUS
 
+mkdir $CORPUS
 cp -r $SCRIPT_DIR/seeds $CORPUS
```

Comment:
I think we should remove the `cp` and pass both `$CORPUS` and the seeds directory to libFuzzer below.